### DISCUSSION TITLE
Update scrollbar width / height

### DISCUSSION THREAD

**Discussion by johnroyer** [rustdesk/rustdesk#5677] [Feature Request]
The width of scroll bar at v1.2.2 is less then 5 pixel. It is too hard to click on it.

![螢幕截圖2023-09-13 12-15-27](https://github.com/rustdesk/rustdesk/assets/114491/96d9250b-9426-4f43-b25f-99e523e1c687)

Hoping the width of scroll bar can be at least 5 pixel or more.

----

Environment:
- Linux (UbuntuMate 22.04 LTS)
- Rustdesk Linux v1.2.2

**Comment by rustdesk** [comment]
@fufesou, also check windows/mac, make sure same to OS.

**Comment by dignow** [comment]
https://github.com/rustdesk/rustdesk/pull/5682

**Comment by johnroyer** [comment]
Scrollbar width has been updated on v1.2.3. Thanks for help.

![2023-10-09-20:44:38-001](https://github.com/rustdesk/rustdesk/assets/114491/a6fbf7ac-4825-4550-906c-93672088a08e)
